src/GraphQl/Resolver/Factory/ItemMutationResolverFactory.php

```diff
@@ -91,43 +91,56 @@ public function __invoke(string $resourceClass = null, string $rootClass = null,
                     throw Error::createLocatedError(sprintf('Item "%s" did not match expected type "%s".', $args['input']['id'], $resourceMetadata->getShortName()), $info->fieldNodes, $info->path);
                 }
             }
-
-            $this->canAccess($this->resourceAccessChecker, $resourceMetadata, $resourceClass, $info, $item, $operationName);
+            $previousItem = \is_object($item) ? clone $item : $item;
 
             $inputMetadata = $resourceMetadata->getGraphqlAttribute($operationName, 'input', null, true);
             $inputClass = null;
             if (\is_array($inputMetadata) && \array_key_exists('class', $inputMetadata)) {
                 if (null === $inputMetadata['class']) {
```

Comment:
@soyuka I don't understand why we expose a mutation if `input_class` is `null`. Shouldn't we just don't register the mutation in this case?

Reply:
From what I remember, disabling the input should disable mutations on this class?